src/selectors.js

```diff
@@ -31,3 +31,9 @@ export const getBy = options => (propertyPath, values) => (state) => {
   if (Array.isArray(values)) return data.filter(d => values.includes(at(d, propertyPath)[0]))
   return data.filter(d => values === at(d, propertyPath)[0])
 }
+
+export const hasKey = options => key => (state) => {
+  const keys = getKeys(options)(state)
+
+  return !!keys.includes(key)
```

Comment:
Is there a reason to use `!!`, I believed `includes` already returns `true` or `false`
I think you can do:
```es6
return keys.includes(key)
```

Reply:
There is no reason this is something I do everytime I need a boolean now.
But you are right in this case. I'll fix that